### PR TITLE
Add index container style overrides

### PR DESCRIPTION
**Pull request by sabymike**
Addresses #16

### CODE REVIEW

File: src/components/AlphabetList/index.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ export const AlphabetList: React.FC<AlphabetListProps> = (props) => {
     data,
     index = DEFAULT_CHAR_INDEX,
     style,
+    indexContainerStyle,
```
Comment (Kieran-McIntyre) [inline src/components/AlphabetList/index.tsx:17]: We need to add `indexContainerStyle` to the `AlphabetListProps` interface. 

You can find this file at `src/components/AlphabetList/types.ts`, and I reckon we can just add 

```ts
indexContainerStyle?: ViewStyle,
```

File: src/components/ListLetterIndex/index.tsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { ISectionData } from "../AlphabetList/types";
 export const ListLetterIndex: React.FC<ListLetterIndexProps> = ({
   sectionData,
   onPressLetter,
+  indexContainerStyle,
```
Comment (Kieran-McIntyre) [inline src/components/ListLetterIndex/index.tsx:10]: Same here, we need to define `indexContainerStyle` in `src/components/ListLetterIndex/types.ts`. We can do:

```ts
indexContainerStyle?: AlphabetListProps["indexContainerStyle"],
```

**Review by Kieran-McIntyre — CHANGES_REQUESTED** [review]
Hey @sabymike, thanks for taking a look at this!
 
PR is looking good, we're just getting a few failing tests due to TypeScript errors. I've added some comments below detailing some changes we'll need to make.

I'd run `yarn run test` before pushing again, just to make sure they're all passing 👍

**Comment by sabymike** [comment]
My apologies, I definitely should have ran the test ahead of time. My Typescript is limited so I guess I didn't expect my change to break any tests. I've pushed an update. Thanks for the tip.

**Comment by Kieran-McIntyre** [comment]
> My apologies, I definitely should have ran the test ahead of time. My Typescript is limited so I guess I didn't expect my change to break any tests. I've pushed an update. Thanks for the tip.

Nice! Thanks for that @sabymike. I've merged this pull request and these changes should now be available in `v2.0.7`. Let me know how you get on 👍

**Pull request merged**